### PR TITLE
(i18n) : integrate latest zh_CN translation

### PR DESCRIPTION
**Pull request by akhabali**
**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: components/components-bigquery/bigquery-definition/src/main/resources/org/talend/components/bigquery/output/BigQueryOutputProperties_zh_CN.properties [hunk 1]
```diff
@@ -8,3 +8,6 @@ property.possiblevalue.DROP_IF_EXISTS_AND_CREATE.displayName=\u5982\u679C\u5B58\
 property.possiblevalue.TRUNCATE.displayName=\u622A\u65AD
 property.possiblevalue.APPEND.displayName=\u8FFD\u52A0
 property.possiblevalue.WRITE_TO_EMPTY.displayName=\u5199\u5165\u7A7A\u8868
+
```
Comment (ypiel-talend) [inline components/components-bigquery/bigquery-definition/src/main/resources/org/talend/components/bigquery/output/BigQueryOutputProperties_zh_CN.properties]: Lots of empty lines

File: components/components-snowflake/components-snowflake-definition/src/main/resources/org/talend/components/snowflake/SnowflakeTableProperties_zh_CN.properties [hunk 2]
```diff
@@ -3,3 +3,9 @@ form.Main.displayName=Snowflake \u8868
 form.Reference.title=Snowflake \u8868
 form.Reference.displayName=Snowflake \u8868
 property.tableName.displayName=\u8868
+
```
Comment (ypiel-talend) [inline components/components-snowflake/components-snowflake-definition/src/main/resources/org/talend/components/snowflake/SnowflakeTableProperties_zh_CN.properties]: Lots of empty lines

**Comment by akhabali** [comment]
Those files are generated by the translating tool.
The empty lines comes from the original files (en). I guess it's normal

File: components/components-snowflake/components-snowflake-definition/src/main/resources/org/talend/components/snowflake/SnowflakeTableProperties_zh_CN.properties (hunk 2 above)
Comment (akhabali) [reply components/components-snowflake/components-snowflake-definition/src/main/resources/org/talend/components/snowflake/SnowflakeTableProperties_zh_CN.properties]: https://github.com/Talend/components/pull/1469#issuecomment-476281882

File: components/components-bigquery/bigquery-definition/src/main/resources/org/talend/components/bigquery/output/BigQueryOutputProperties_zh_CN.properties (hunk 1 above)
Comment (akhabali) [reply components/components-bigquery/bigquery-definition/src/main/resources/org/talend/components/bigquery/output/BigQueryOutputProperties_zh_CN.properties]: https://github.com/Talend/components/pull/1469#issuecomment-476281882

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/23/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/23/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Aakhabali%2Fi18n_zh_CN)

**Pull request merged**